### DISCUSSION TITLE
Stack in /app/index.js renders nothing

### DISCUSSION THREAD

**Discussion by thecoorum** [expo/router#147] [Q&A] [answered]
Hey! I'm playing around with this awesome library but met with an issue while doing it. As entry file doesn't contain any providers anymore I found that we can place all providers into `/app/_layout.js` and use `Children` in it. But when I'm trying to render `Stack` in `/app/index.js` it renders nothing. Any ideas?

`/` structure
```
/app
  /home
    index.js
  _layout.js
  index.js
```

`/app/_layout.js`
```
...
<Providers>
  <Children />
</Providers>
```
`/app/index.js` (`initialized` and `fontsLoaded` are `true` at some point so after `Skeleton` component disappears I can see only background screen)
```
<View style={sx({ flex: 1, backgroundColor: '$background' })}>
  <StatusBar style="light" />
  {initialized && fontsLoaded ? (
    <Stack
      screenOptions={{
        headerShown: false
      }}
    >
      <Stack.Screen name="home" />
    </Stack>
  ) : (
    <Skeleton />
  )}
</View>
```

**Comment by EvanBacon** [comment, marked as answer]
You cannot use `<Stack />` directly in a route that isn't `_layout`. You need to move this logic to `_layout` and treat `app/index` as a screen.

I opened an internal issue to add a warning for this.
